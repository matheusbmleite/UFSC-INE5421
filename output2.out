{
    "init_state": [
        "x"
    ],
    "final_states": [
        [
            "x",
            "y"
        ],
        [
            "x",
            "z"
        ]
    ],
    "type": "automaton",
    "states": [
        [
            "x",
            "y"
        ],
        [
            "x",
            "z"
        ],
        [
            "x"
        ]
    ],
    "alphabet": [
        "1",
        "0"
    ],
    "transitions": {
        "['x', 'y']": {
            "1": [
                "x",
                "z"
            ],
            "0": [
                "x",
                "y"
            ]
        },
        "['x', 'z']": {
            "1": [
                "x"
            ],
            "0": [
                "x",
                "y"
            ]
        },
        "['x']": {
            "1": [
                "x"
            ],
            "0": [
                "x",
                "y"
            ]
        }
    }
}