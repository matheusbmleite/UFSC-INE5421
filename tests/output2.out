{
    "alphabet": [
        "0",
        "1"
    ],
    "type": "automaton",
    "transitions": {
        "['z', 'x']": {
            "0": [
                "y",
                "x"
            ],
            "1": [
                "x"
            ]
        },
        "['y', 'x']": {
            "0": [
                "y",
                "x"
            ],
            "1": [
                "z",
                "x"
            ]
        },
        "['x']": {
            "0": [
                "y",
                "x"
            ],
            "1": [
                "x"
            ]
        }
    },
    "init_state": [
        "x"
    ],
    "states": [
        [
            "x"
        ],
        [
            "y",
            "x"
        ],
        [
            "z",
            "x"
        ]
    ],
    "final_states": [
        [
            "y",
            "x"
        ],
        [
            "z",
            "x"
        ]
    ]
}